scripts/generate_error_code.hs

```diff
@@ -175,6 +175,11 @@ internalErrors =
       , eiCode = 320
       , eiDesc = "Generic impossible error."
       }
+    , ErrorItem
+      { eiLabel = "negative_seconds_outside_err"
```

Comment:
I usually start the internal error with `internal_*`, may not be necessary but it help when reading the code. I've approved anyway.

Reply:
Nice catch, thanks! Fixed.